src/seedu/addressbook/AddressBook.java

```diff
@@ -1077,15 +1100,26 @@ private static boolean isPersonEmailValid(String email) {
      */
     private static String getUsageInfoForAllCommands() {
         return getUsageInfoForAddCommand() + LS
+        		+ getUsageInfoForSortCommand() + LS
                 + getUsageInfoForFindCommand() + LS
                 + getUsageInfoForViewCommand() + LS
                 + getUsageInfoForDeleteCommand() + LS
                 + getUsageInfoForClearCommand() + LS
                 + getUsageInfoForExitCommand() + LS
                 + getUsageInfoForHelpCommand();
     }
-
     /**
+     * Builds string for showing 'sort' command usage instruction
```

Comment:
good to see you have updated the header comments and help on this one.

please take note, you have added a new functionality here, hence you need to update the tests as well!!